drivers/sht3x/sht3x.c

```diff
@@ -0,0 +1,442 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_sht3x
+ * @brief       Device Driver for Sensirion SHT30/SHT31/SHT35 Humidity and Temperature Sensors
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ * @file
+ */
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+#include "log.h"
+
+#include <errno.h>
+#include <string.h>
+
+#include "sht3x.h"
+#include "xtimer.h"
+
+#define ASSERT_PARAM(cond) \
+    if (!(cond)) { \
+        DEBUG("[sht3x] %s: %s\n", \
+              __func__, "parameter condition (" #cond ") not fulfilled"); \
+        assert(cond); \
+    }
+
+#define DEBUG_DEV(f, d, ...) \
+        DEBUG("[sht3x] %s dev=%d addr=%02x: " f "\n", \
+              __func__, d->i2c_dev, d->i2c_addr, ## __VA_ARGS__);
+
+#define ERROR_DEV(f, d, ...) \
+        LOG_ERROR("[sht3x] dev=%d addr=%x: " f "\n", \
+                  d->i2c_dev, d->i2c_addr, ## __VA_ARGS__);
+
+/** SHT3x common commands */
+#define SHT3X_STATUS_CMD               0xF32D
+#define SHT3X_CLEAR_STATUS_CMD         0x3041
+#define SHT3X_RESET_CMD                0x30A2
+#define SHT3X_FETCH_DATA_CMD           0xE000
+#define SHT3X_HEATER_OFF_CMD           0x3066
+
+/** SHT3x status register flags */
+#define SHT3X_STATUS_REG_MASK   (0xbc13)
+#define SHT3X_STATUS_REG_CRC    (1 << 0)
+#define SHT3X_STATUS_REG_CMD    (1 << 1)
+
+/** SHT3x measurement period times in us */
```

Comment:
@maribu Could you please revise these types of change requests? I went through all other other driver implementations. Nowhere else are doxygen-style C files documented, with the exception of the file header. It would make a lot of work to comment the C file in a doxygen-compliant way without any addition of information.

Reply:
> I think you should add @brief (e.g. /** @brief SHT3x ... */).

As I remember, for a short comment it makes no difference in doxygen whether just ```/** */``` is used or ```/** @brief */```